custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssueNavigator.kt

```diff
@@ -65,6 +86,59 @@ class CloudIssueNavigator(
         )
     }
 
+    override fun clickOnDetails() {
+        driver.wait(
+            elementToBeClickable(By.xpath("//*[@class = 'show-filter-details']"))
+        )
+            .click()
+    }
+
+    override fun checkSubscriptions() {
+        driver.wait(
+            and(
+                visibilityOfElementLocated(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[@class = 'search-owner']")),
+                visibilityOfElementLocated(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[contains(text(), 'Owned by')]")),
+                visibilityOfElementLocated(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[contains(text(), 'Edit permissions')]")),
+                visibilityOfElementLocated(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[contains(text(), 'Edit permissions')]"))
+            )
+        )
+    }
+
+    override fun newSubscription() {
+        driver.wait(
+            elementToBeClickable(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[@class='trigger-dialog' and contains(text(), 'New subscription')]"))
+        )
+            .click()
+    }
+
+    override fun waitForFilterSubscriptionLoad() {
+        filterSubscriptionFalliblePage.waitForPageToLoad()
+    }
+
+    override fun subscribe() {
+        driver.wait(
+            elementToBeClickable(By.id("filter-subscription-submit"))
+        )
+            .click()
+
+        driver.wait(
+            invisibilityOfElementLocated(By.id("filter-subscription-submit"))
+        )
+    }
+
+    override fun manageSubscriptions(): ViewSubscriptions {
+        driver.wait(
+            elementToBeClickable(By.xpath("//*[@class = 'manage-links']//a[. = 'Manage subscriptions']"))
+        )
+            .click()
+        val cloudViewSubscriptions = CloudViewSubscriptions(jira)
+        cloudViewSubscriptions.waitForLoad()
+        return cloudViewSubscriptions
+    }
+
+    override fun closeSubscriptionAddedPopup(){
```

Comment:
why do we need this method?

Reply:
We have different workflow for DC and Cloud.
Cloud don't have such functionality, but this is used in DC